@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/engine/fn_engine.sqf

```diff
@@ -71,6 +81,7 @@ _engBaseNG = _engIdleNG + (_engFlyNG - _engIdleNG) * _engThrottle;
 //Np
 _engBaseNP = _engIdleNP + (_engFlyNP - _engIdleNP) * _engThrottle;
 
+
```

Comment:
Please can you remove the additional space here?

Reply:
Complete